test/ejabberd_config_SUITE_data/ejabberd.no_listeners.gd.node1_v1.cfg

```diff
@@ -0,0 +1,44 @@
+{loglevel, 3}.
+{hosts, ["localhost", "mim.localhost", "c1-mim.localhost"] }.
+%% To avoid eaddinuse, we don't define any listeners
```

Comment:
eaddrinuse ?

Reply:
error code, stands for "address in use error".